packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -99,15 +130,53 @@ export function Menu({
           Math.min(options.length - 1, highlightedIndex + indexChange),
         );
       }
+      if (menuDiv.current && menuItemRefs.current[highlightedIndex]) {
+        scrollDownIfTooLow(
+          menuDiv.current,
+          menuItemRefs.current[highlightedIndex],
+        );
+      }
     });
 
+    function scrollDownIfTooLow(
```

Comment:
I think we should be able to generalize this into a single method with `scrollUpIfTooHigh`.

Reply:
Totally, DRY'd up these methods into one!